lib/src/main/java/macrobase/ingest/FastDataFrameCSVLoader.java

```diff
@@ -0,0 +1,118 @@
+package macrobase.ingest;
+
+import macrobase.datamodel.DataFrame;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.*;
+
+import static macrobase.datamodel.Schema.ColType;
+
+/**
+ * Not as robust as the standard csv loader but 10x faster
```

Comment:
Not as robust meaning what?

What if you just wrap this, catch exceptions, then call the Apache one if this fails.